Common/src/geometry/CPhysicalGeometry.cpp

```diff
@@ -3913,22 +3880,18 @@ void CPhysicalGeometry::LoadLinearlyPartitionedVolumeElements(CConfig        *co
     }
   }
 
-
   /*--- Reduce the global counts of all element types found in
    the CGNS grid with all ranks. ---*/
 
```

Comment:
I may be a bit picky here, but why are multiple calls to Allreduce used here? I know this happens in quite a few places in the code, but I have never understood why this is done instead of a single call to Allreduce with an array.

Reply:
I imagine whoever did it like that was thinking of readability of having multiple descriptive name vs one generic array.
We could have special types for that kind of data to strike a compromise, but since this particular case is not in the hot path of the code I would leave like that.

Reply:
Agree